.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Recipe
+about: This template makes sure relevant information is contained in a new recipe pull request.
+title: [Dataset Name]
+labels: ''
+assignees: ''
+
+---
+<!-- 
+👋 Thanks for opening a PR! This template describes the relevant information for a new recipe pull request.
+
+Appropriate files:
+Do you have a meta.yaml and a recipe.py in this PR? If you don't know what those are, please check out the sandbox https://mybinder.org/v2/gh/pangeo-forge/sandbox/binder or https://pangeo-forge.readthedocs.io/en/latest/introduction_tutorial/intro_tutorial_part3.html
+
+--!>
+
+## Dataset
+<!-- Please reference the proposed issue for this dataset.  If you haven't created one yet, you can do so with this template https://github.com/pangeo-forge/staged-recipes/issues/new/choose --!>
```

Comment:
Recipe PRs do not require a linked Issue, they can begin directly with a PR (of course if there _is_ a linked issue, we'd want to be aware of that)

Reply:
Got it. I updated it to not require an issue, but request that info in the PR. Make sense or should I totally forget about the issue template?